teuthology/config.py

```diff
@@ -33,12 +33,13 @@ def __init__(self, yaml_path=None):
             self._conf = dict()
 
     def load(self, conf=None):
-        if conf:
+        if conf is not None:
             if isinstance(conf, dict):
                 self._conf = conf
-            else:
+                return
+            elif conf:
                 self._conf = yaml.safe_load(conf)
-            return
+                return
```

Comment:
Why duplicate returns?

Reply:
nit

Reply:
I don't quite understand the question; there are two different return statement for two different conditions, and if neither condition is satisfied the method continues

Reply:
in both cases it returns nothing, there is no sense to pull return statements into conditionals of is dict type checking.

Reply:
I still don't understand the concern here. Are you saying you think there's a bug here, or some sort of style issue?

Reply:
style, no bug, I've got concern, because it looks unnecessary change and it does not improve anything for my taste, sorry about that, I guess, nit means you can ignore it